team-cost-reporter/cloudcheckr.py

```diff
@@ -167,6 +169,9 @@ def convert(cost_data, tag, debug=False):
     return_data['Total'] = total
     return_data['Max'] = max
     return_data['Min'] = min
-    return_data['Average'] = total / count
```

Comment:
Was this what was causing the problem?

Reply:
It was causing some problem while troubleshooting. I don't know what the root problem actually was. I needed to update to Python3 in order to troubleshoot, and the problem went away.